wurst/systems/experience/Experience.wurst

```diff
@@ -32,11 +37,31 @@ init
     for i = 0 to PLAYER_NEUTRAL_PASSIVE - 1
         players[i].setHandicapXP(0)
 
+    doPeriodically(ANIMATION_PERIOD) (cb) ->
+        for troll in udg_trolls
+            if troll.getMana() < 1 and troll.isAlive()
+                let owner = troll.getOwner()
+                if owner.getGold() < 1

```

Comment:
Why does the dying troll need both 0 heat and mana? Shouldn't it only require one of the two?

Reply:
Good call that would have been a bug. Fixed it